data/json/recipes/recipe_others.json

```diff
@@ -3252,7 +3252,7 @@
       [ "stick", 2 ],
       [ "broom", 2 ],
       [ "mop", 2 ],
-      [ "2x4", 1 ],
+      [ "2x4", 2 ],
```

Comment:
So, it takes only half as many two by fours, but it still takes 2 brooms, sticks, mops or cues? What exactly is the logic here? (I can just craft a stick into a two by four and use it instead). And if it takes less wood, shouldn't it take less of the other other stuff as well?

If it takes less components, the item should be smaller and have less weight.